pkgbuild/README.md

```diff
@@ -1,9 +1,18 @@
 # pkgbuild mac installers
 
+## Prerequisites
+
+The generation of the OpenJDK installer depends on an open-source utility tool [Packages](http://s.sudre.free.fr/Software/Packages/about.html).
+The application must be installed before running the installer generation task.
+
+```bash
+brew cask install packages
+```
+
 ## Example Usage
```

Comment:
Should explain how to override the vendor and that one might shoot themselves in the foot by only overriding vendor and not identifier, too. That could result in a identifier like `net.acme` even if you don't own `net.acme`.